descartes_planner/include/descartes_planner/dense_planner.h

```diff
@@ -23,15 +23,15 @@ class DensePlanner: public descartes_core::PathPlannerBase
 
   virtual bool initialize(descartes_core::RobotModelConstPtr &model);
   virtual bool setConfig(const descartes_core::PlannerConfig& config);
-  virtual void getConfig(descartes_core::PlannerConfig& config);
+  virtual void getConfig(descartes_core::PlannerConfig& config) const;
   virtual bool planPath(const std::vector<descartes_core::TrajectoryPtPtr>& traj);
```

Comment:
Omitting 'planPath' from being a const accessible method would make the class unusable